package.json

```diff
@@ -62,7 +62,10 @@
     "graceful-fs": "^4.1.11",
     "ini": "^1.3.5",
     "ink": "^0.5.1",
+    "ink-select-input": "^2.0.0",
+    "ink-spinner": "^2.0.0",
     "ink-table": "^1.0.3",
+    "ink-text-input": "^2.0.0",
     "jsx-transform": "^2.4.0",
```

Comment:
If you're gonna use `figures`, please make sure it ends up in our dependencies!

Reply:
Woops—it's a dependency of `ink-spinner` and I was tinkering... I'll add it!